gateway/api/migrations/0014_runtimejob.py

```diff
@@ -7,7 +7,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ("api", "0013_program_groups_catalogentry"),
+        ("api", "0012_jobconfig_python_version"),
```

Comment:
Removing migrations or changing order of migration will break existing databases. If we want to remove code, related to models we need to remove models and generate new migration which will delete tables from datase.

Long story short: migrations are always append only :)

Reply:
oops! Just saw your last comment about not published. 

I think it is still better to append migrations, if it is not hard to do

Reply:
Solved in [674a79c](https://github.com/Qiskit-Extensions/quantum-serverless/pull/1276/commits/674a79c9350452c8bc97fef38866a5c8f933dc3b)